src/views/v2/emails/template-parts/body/order/ticket-totals/ticket-title.php

```diff
@@ -23,7 +23,7 @@
  * @var \WP_Post       $order              The order object.
  */
 
-$ticket_title = empty( $ticket['title'] ) ? '' : $ticket['title'];
+$ticket_title = get_the_title( $ticket );
```

Comment:
If we're already saving the ticket title in the `$order` object, let's make use of that to make it more performant, rather than re-requesting the title with `get_the_title`